DATA/EQUIPMENT/weapon_equip.ini

```diff
@@ -35668,7 +35674,9 @@ ids_info = 461445
 ; Mass: $mass
 ; Integrity: $hitpoints
 ; \
-; This neutron-based weapon is a fast development by ex-engineers from RGS who worked on this weapon shortly before the big wave of layoffs on alster began. Without this first symbol of revolt the unioners maybe never went strong as they are today. neutron weapons deal +25% damage to molecular shields but weak against positron shields
+; This neutron-based weapon is a fast development by ex-engineers from RGS who worked on this weapon shortly before the big wave of layoffs on alster began. Without this first symbol of revolt the unioners maybe never went strong as they are today.

```

Comment:
"Unioniers" is a name and should be capitalized

Reply:
I'm aware of these issues but I am fixing them when I re-make the Rheinland pirate weapon descriptions from the ground up, rather than putting in the work to polish them now.